### PR TITLE
kubevirt: drop kubernetes and kubevirt dependencies

### PR DESCRIPTION
**Pull request by pkliczewski**
It is common practice in virt-who not to rely on external dependencies.
This PR removes dependency on kubevirt-python and the kubernetes-python.

Fixes: https://bugzilla.redhat.com/1656265
Fixes: https://bugzilla.redhat.com/1684499

### CODE REVIEW

File: virtwho/virt/kubevirt/client.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            key_file=key_file
+        )
+        
+        self.host = cfg.host
+
+    def get_nodes(self):
+        return self._request('/api/v1/nodes')
+
+    def get_vms(self):
+        return self._request('/apis/kubevirt.io/v1alpha3/virtualmachineinstances')
```
Comment (vatsalparekh) [inline virtwho/virt/kubevirt/client.py]: Suggestion: hard-coding of endpoint have bitten us in past, so I've a feeling of following the dynamic method and get `preferred_version` of endpoint wherever we can.

Comment (pkliczewski) [reply virtwho/virt/kubevirt/client.py]: Good point, will add version discovery.

**Comment by pkliczewski** [comment]
@wottop Please review. Build failure is not related to this change.

**Comment by wottop** [comment]
buildbot: retest this please

**Comment by wottop** [comment]
ack

**Pull request merged**